src/iff/iff2.c

```diff
@@ -239,7 +237,7 @@ main(int argc, char *argv[], char *envp[])
 		parentport = ntohs(parentsock_port);
 
 	pbs_errno = 0;
-	err = tcp_send_auth_req(sock, parentport, pwent->pw_name);
+	err = tcp_send_auth_req(sock, parentport, pwent->pw_name, AUTH_RESVPORT_NAME, getenv(PBS_CONF_ENCRYPT_METHOD));
```

Comment:
something like RESVPORT_AUTH will be more readable compared to AUTH_RESVPORT_NAME

Reply:
I feel verything in the auth API starts with AUTH, so it actually makes sense to have it start with AUTH_. This is in sync with what the others are being called for example. AUTH_GSS_NAME

Reply:
It's the postfix `_NAME` that causes confusion. That makes it sound more like a variable than a macro. It would have been better if you keep it as AUTH_RESVPORT and AUTH_GSS.

Reply:
@nithinj I understand your confusion, but for me AUTH_RESVPORT looks like int value in some enum/macro (like earlier we had), where here I wanted to explicitly say its string and its name of auth method, that's why I added `_NAME` in it. And if you see `AUTH_RESVPORT_NAME` as variable then its kinda `const char *` variable right, just that I used `#define` instead making it const char *. HTH!